machina/templates/machina/partials/profil_link.html

```diff
@@ -0,0 +1,10 @@
+{% load forum_member_tags %}
+
+{% if poster %}
+    {% url 'forum_member:profile' poster.id as profil_url %}
+    {% with username=poster|forum_member_display_name %} 
+        <a href="{{profil_url }}" title="title">
+            {% if has_bold %}<b>{%endif%}{{ username }}{% if has_bold %}</b>{%endif%}
```

Comment:
I don't like the `has_bold` parameter name.
What do you think about `highlight` ?